Engine/Client/Src/Application/Application.cpp

```diff
@@ -119,6 +119,9 @@ void Application::_handleJoinRoomPacket(ABINetwork::UDPPacket packet)
         return;
     }
     _client = room;
+    ABINetwork::sendPacketInit(_client);
+    ABINetwork::sendPacketInit(_client);
+    ABINetwork::sendMessages(_client);

```

Comment:
Is this sendMessages really usefull ?

Reply:
Yes really because I've to send it directly